packages/core/src/components/ui/Button/ButtonSignIn/ButtonSignIn.tsx

```diff
@@ -3,14 +3,16 @@ import { LinkButton } from '@faststore/ui'
 import styles from 'src/components/ui/Button/button.module.scss'
 import Icon from 'src/components/ui/Icon'
 import { useSession } from 'src/sdk/session'
+import { useCart } from '../../../../sdk/cart/index'
 
 const ButtonSignIn = () => {
+  const { isValidating, id } = useCart()
   const { person } = useSession()
 
   return (
     <LinkButton
       data-fs-button-signin-link
-      href={person?.id ? '/account' : '/login'}
+      href={person?.id ? `/account?orderFormId=${id}` : `/login?orderFormId=${id}`}
```

Comment:
out of curiosity, why should we forward the `orderFormId` here?

Reply:
We must pass this parameter to do something similar to what the cart page does and update the orderform of the session if it is different.
Ex: Customer has a cart set up and accesses my account. In this scenario, if gnt follows the flow of replicating the session and then validating whether there is a divergence, the customer would lose the cart. That way, when the customer brings this orderform to the portal, I can insert it in the session, preventing the customer from losing his cart.

Reply:
and thanks for all the insight at this PR ❤️